pageserver/src/walreceiver.rs

```diff
@@ -105,10 +106,22 @@ async fn walreceiver_main(
         (end_of_wal >> 32),
         (end_of_wal & 0xffffffff)
     );
-    let startpoint = tokio_postgres::types::Lsn::from(startpoint);
-    let mut physical_stream = rclient
-        .start_physical_replication(None, startpoint, None)
-        .await?;
+
+    let startpoint = PgLsn::from(startpoint);
+
+    // FIXME: unimplemented : SLOT name; TIMELINE id
+    // FIXME: these options are used in the tokio-postgres test code; are they useful to us?
```

Comment:
The "publication" stuff is related to logical replication, so that's not relevant to us (we're odoing physical). I'm not sure what the `proto_version` is, but I presume it's also related to logical replication.

Reply:
OK, I'll remove those comments. Thanks!